worker/backend/container.go

```diff
@@ -0,0 +1,139 @@
+package backend
+
+import (
+	"io"
+	"time"
+
+	"code.cloudfoundry.org/garden"
+)
+
+type Container struct{}
+
+var _ garden.Container = (*Container)(nil)
+
+func (c *Container) Handle() (handle string) { return }
+
+// Stop stops a container.
+//
+// If kill is false, garden stops a container by sending the processes running inside it the SIGTERM signal.
+// It then waits for the processes to terminate before returning a response.
+// If one or more processes do not terminate within 10 seconds,
+// garden sends these processes the SIGKILL signal, killing them ungracefully.
+//
+// If kill is true, garden stops a container by sending the processing running inside it a SIGKILL signal.
+//
+// It is possible to copy files in to and out of a stopped container.
+// It is only when a container is destroyed that its filesystem is cleaned up.
+//
+// Errors:
+// * None.
+func (c *Container) Stop(kill bool) (err error) { return }
+
+// Returns information about a container.
+func (c *Container) Info() (info garden.ContainerInfo, err error) { return }
+
+// StreamIn streams data into a file in a container.
+//
+// Errors:
+// *  TODO.
+func (c *Container) StreamIn(spec garden.StreamInSpec) (err error) { return }
+
+// StreamOut streams a file out of a container.
+//
+// Errors:
+// * TODO.
+func (c *Container) StreamOut(spec garden.StreamOutSpec) (readCloser io.ReadCloser, err error) { return }
+
+// Returns the current bandwidth limits set for the container.
+func (c *Container) CurrentBandwidthLimits() (limits garden.BandwidthLimits, err error) { return }
+
+// Returns the current CPU limts set for the container.
+func (c *Container) CurrentCPULimits() (limits garden.CPULimits, err error) { return }
+
+// Returns the current disk limts set for the container.
+func (c *Container) CurrentDiskLimits() (limits garden.DiskLimits, err error) { return }
+
+// Returns the current memory limts set for the container.
+func (c *Container) CurrentMemoryLimits() (limits garden.MemoryLimits, err error) { return }
+
+// Map a port on the host to a port in the container so that traffic to the
+// host port is forwarded to the container port. This is deprecated in
+// favour of passing NetIn configuration in the ContainerSpec at creation
+// time.
+//
+// If a host port is not given, a port will be acquired from the server's port
+// pool.
+//
+// If a container port is not given, the port will be the same as the
+// host port.
+//
+// The resulting host and container ports are returned in that order.
+//
+// Errors:
+// * When no port can be acquired from the server's port pool.
```

Comment:
if you're wondering where these comments came from: they're straight from `garden` - we can totally remove them, but it seemed helpful for me to have them while implementing the methods. totally fine w/ removing though